### ISSUE TITLE
Anterior drawer test positive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11151]
**Preferred term label:**
Anterior drawer test positive

**Synonyms**

**Definition (free text, please give PubMed ID)**
Anterior pull on the calcaneus with one hand while stabilizing the end of the tibia/fibula with the other hand results in >1cm of anterior translation of the foot compared to the healthy, contralateral ankle.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ankle sprain

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Just to avoid confusion: there are two such tests, one for knee (https://www.physio-pedia.com/Anterior_Drawer_Test_of_the_Knee) and one for ankle. 
Do we want a specific Ankle term - Positive anterior drawer test of the ankle?

**Comment by MickeySegal** [comment]
Yes, a specific foot/ankle term.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001342
- label: Positive anterior drawer test
- parents: [Abnormal ankle physiology (HP:0034674)](https://hpo.jax.org/app/browse/term/HP:0034674)
- pmid: PMID:36849957
- def: A test for ankle instability in which an anterior pull on the calcaneus with is applied with one hand while stabilizing the end of the tibia/fibula with the other hand. The test is considered positive (abnormal) if it results in more than one cm of anterior translation of the foot compared to the healthy, contralateral ankle.
- comment: A positive anterior drawer test indicates possible damage or tearing of the anterior cruciate ligament (ACL) or the anterior talofibular ligament (ATFL).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11151
- synonyms: Anterior drawer test positive(NONE)
